src/lib/state/DY.Lib.State.Tagged.fst

```diff
@@ -40,45 +40,50 @@ type local_bytes_state_predicate {|crypto_invariants|} = {
   ;
 }
 
-let split_local_bytes_state_predicate_func {|crypto_invariants|} : split_predicate_input_values = {
+let split_local_bytes_state_predicate_func {|crypto_invariants|} : split_function_input_values = {
   tagged_data_t = trace & principal & state_id & bytes;
+
+  tag_set_t = string;
   tag_t = string;
-  encoded_tag_t = string;
+  is_disjoint = default_disjoint;
+  tag_belong_to = (fun dtag tag -> dtag = tag);
+  cant_belong_to_disjoint_sets = (fun dtag tag1 tag2 -> ());
+
   raw_data_t = trace & principal & state_id & bytes;
+  output_t = prop;
+
+  default_global_fun = (fun tr prin sess_id sess_content -> False);
```

Comment:
this should move after `global_fun`. (same as for `DY.Lib.Event.Typed`)

Reply:
Good catch, I changed that!